crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1907,6 +1941,22 @@ impl fmt::Display for FakeID {
     }
 }
 
+impl Default for AdapterInitConfig {
+    fn default() -> Self {
+        Self {
+            additional_mapping: BTreeMap::new(),
+            account_names: BTreeSet::new(),
+            protocol_config: ProtocolConfig::get_for_max_version_UNSAFE(),
+            is_simulator: false,
+            custom_validator_account: false,
+            reference_gas_price: None,
+            default_gas_price: None,
+            flavor: None,
+            offchain_config: None,
+        }
+    }
+}
+
```

Comment:
If `ProtocolConfig` has a `Default`, then we could just slap a `#[derive(Default)]` on `AdapterInitConfig`.

Reply:
while that's fair, i think i'll keep it like this just so it's explicitly clear the `unsafe` bit

Reply:
I would say that `ProtocolConfig` does not have a reasonable notion of "Default" so +1 to Will's comment about it being explicit.